src/Core/ProtocolDefines.h

```diff
@@ -112,12 +112,14 @@ static constexpr auto DBMS_MIN_REVISON_WITH_PARALLEL_BLOCK_MARSHALLING = 54478;
 
 static constexpr auto DBMS_MIN_REVISION_WITH_VERSIONED_CLUSTER_FUNCTION_PROTOCOL = 54479;
 
+static constexpr auto DBMS_MIN_REVISION_WITH_COMPRESSED_LOGS_PROFILE_EVENTS = 54480;
```

Comment:
Maybe `TableColumns` also should be compressed (https://github.com/ClickHouse/ClickHouse/issues/75719)?

Reply:
Added.